packages/types-core/test/project.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020-2023 SubQuery Pte Ltd authors & contributors
+// SPDX-License-Identifier: GPL-3.0
+
+// import {ProjectManifestV1_0_0} from '@subql/common';
+// import {ProjectManifestV1_0_0Impl} from '@subql/common-substrate';
+// import {ProjectManifestImpls} from './ProjectManifestVersioned'
+// import {SubstrateDatasource} from '@subql/types';
+
+//
+// const manifest: ProjectManifestV1_0_0<SubstrateDatasource> = {
```

Comment:
Should users be using a specific version here? I thought it was `SubstrateProject`?

Reply:
Was a draft, this can be removed now